infra/token-transfer-client/src/components/pages/WithdrawalHistory.js

```diff
@@ -44,6 +44,9 @@ const WithdrawalHistory = props => {
     )
   }
 
+  const unlockDate = props.user.employee
```

Comment:
Super nit: in order to avoid code duplication, might be worth putting this code in a util function that takes a user as arg and returns the unlock date.

Reply:
Good idea 👍